go/plugins/googlegenai/models.go

```diff
@@ -75,40 +77,47 @@ var (
 				"gemini-1.5-flash-8b-001",
 			},
 			Supports: &gemini.Multimodal,
+			Stage:    ai.ModelStageStable,
 		},
 		gemini20Flash: {
 			Label: "Gemini 2.0 Flash",
 			Versions: []string{
 				"gemini-2.0-flash-001",
 			},
 			Supports: &gemini.Multimodal,
+			Stage:    ai.ModelStageStable,
 		},
 		gemini20FlashLite: {
 			Label: "Gemini 2.0 Flash Lite",
 			Versions: []string{
 				"gemini-2.0-flash-lite-001",
 			},
 			Supports: &gemini.Multimodal,
+			Stage:    ai.ModelStageStable,
 		},
 		gemini20FlashLitePrev: {
 			Label:    "Gemini 2.0 Flash Lite Preview 02-05",
 			Versions: []string{},
 			Supports: &gemini.Multimodal,
+			Stage:    ai.ModelStageStable,
 		},
 		gemini20ProExp0205: {
 			Label:    "Gemini 2.0 Pro Exp 02-05",
 			Versions: []string{},
 			Supports: &gemini.Multimodal,
+			Stage:    ai.ModelStageStable,
 		},
 		gemini20FlashThinkingExp0121: {
 			Label:    "Gemini 2.0 Flash Thinking Exp 01-21",
 			Versions: []string{},
 			Supports: &gemini.Multimodal,
+			Stage:    ai.ModelStageStable,
```

Comment:
Anything marked with "exp" or "preview" is experimental so should be `ai.ModelStageUnstable`.